models/articles.model.js

```diff
@@ -34,4 +34,23 @@ exports.fetchArticles = () => {
         .then(({rows}) => {
             return rows
         })
-}
\ No newline at end of file
+}
+
+exports.fetchCommentsByArticleId = (id) =>{
+    // const currentArticleIdNums = db.query('SELECT article_id FROM articles;')
+    // console.log(currentArticleIdNums)
+
+    return db
+        .query(
+            `SELECT * FROM comments
+            WHERE article_id=$1;`, [id]) 
+        .then(({rows}) => {
+                if (rows.length === 0) {
+                    return Promise.reject({ 
+                        status: 404, 
+                        message: `No comments found. Article ID ${id} does not exist.`
+                    })
+                }
```

Comment:
If an empty array comes back its not an error it just means that there is no comments with that article which you would want to send back to the client.